src/chrome/content/rules/Elster.de.xml

```diff
@@ -0,0 +1,16 @@
+<ruleset name="Elster.de">
+	<target host="www.elster.de" />
+	<target host=    "elster.de" />
+	<target host="www.elsteronline.de" />
+	<target host=    "elsteronline.de" />
+
+	<test url="https://elster.de/" />
+	<test url="https://elsteronline.de/" />
+
+	<rule from="^https?://elster\.de/"
+		to="https://www.elster.de/" />
+	<rule from="^https?://elsteronline\.de/"
+		to="https://www.elsteronline.de/" />
```

Comment:
I guess lines 10-13 (and line 7-8 as well) are not required because https://elster.de/ and https://elsteronline.de/ display just fine. Some website ([example](https://pbebank.com/)) requires url rewrite (to add www) because without it, the link is either unreachable or SSL error.